gen/dcos-config.yaml

```diff
@@ -447,8 +447,6 @@ package:
       {
         "PATH": "/usr/bin:/bin",
         "SHELL": "/usr/bin/bash",
-        "LD_LIBRARY_PATH": "/opt/mesosphere/lib",
```

Comment:
I'm not sure we can remove this yet in 1.9 (although I wish we could), since every universe framework scheduler except Spark depends on host libmesos, and each of these schedulers runs as an executor, using these executor-environment vars.

Before we merge this, I'd like to see at least Cassandra/Kafka/HDFS, Jenkins, and MoM running (and successfully registered with Mesos) on a cluster from this PR; and an EE strict-mode cluster with the EE versions of those frameworks (including MoM-EE) successfully registered. It'd be great if this was an automated integration test
If that all still works, we can drop LD_LIBRARY_PATH from here. If not, we'll need to wait another release.
Even then, all the rest of the universe frameworks not officially supported by Mesosphere are also likely to break, so we'd need to notify the developers (Do we have a dev@?)